src/components/Header.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import Search from './blocks/Search';
+import Logo from '../components/blocks/Logo';
+import { useDispatch } from "react-redux";
+import { openFormForCreateOrUpdateOrDelete } from "../store/reducers/movies";
+
+const Header: React.FC = () => {
+    const dispatch = useDispatch()
+
+    const openFormforCreate = () => dispatch(openFormForCreateOrUpdateOrDelete())
```

Comment:
why don't you use `openForm` instead of `openFormForCreateOrUpdateOrDelete`?

Reply:
fixed it)